dea/apps/index_from_tar.py

```diff
@@ -6,14 +6,14 @@
 from dea.index import from_yaml_doc_stream
 
 
-def from_tar_file(tarfname, index, mk_uri, mode, **kwargs):
+def from_tar_file(tarfname, index, protocol, mk_uri, mode, **kwargs):
```

Comment:
This is not needed, the idea is that `mk_uri` captures whatever extra parameters be it `protocol` or credentials for url shortener.